presto-product-tests/src/main/java/io/prestosql/tests/hive/TestHiveStorageFormats.java

```diff
@@ -454,99 +454,49 @@ public void testTimestampCreatedFromPresto(StorageFormat storageFormat)
         String tableName = "test_timestamp_" + storageFormat.getName().toLowerCase(Locale.ENGLISH);
         setupTimestampData(tableName, storageFormat);
 
-        // commenting out pre-epoch timestamps until https://github.com/prestosql/presto-hive-apache/pull/17 becomes available
         List<TimestampAndPrecision> data = ImmutableList.of(
-//                new TimestampAndPrecision(
-//                        "MILLISECONDS",
-//                        "1967-01-02 12:34:56.123",
-//                        ImmutableMap.of(
-//                                "MILLISECONDS", "1967-01-02 12:34:56.123",
-//                                "MICROSECONDS", "1967-01-02 12:34:56.123",
-//                                "NANOSECONDS", "1967-01-02 12:34:56.123")),
                 new TimestampAndPrecision(
                         "MILLISECONDS",
                         "2020-01-02 12:34:56.123",
                         ImmutableMap.of(
                                 "MILLISECONDS", "2020-01-02 12:34:56.123",
                                 "MICROSECONDS", "2020-01-02 12:34:56.123",
                                 "NANOSECONDS", "2020-01-02 12:34:56.123")),
-//                new TimestampAndPrecision(
-//                        "MILLISECONDS",
-//                        "1967-01-02 12:34:56.1234",
-//                        ImmutableMap.of(
-//                                "MILLISECONDS", "1967-01-02 12:34:56.123",
```

Comment:
what are these cases going away now?

Reply:
These are waiting for https://github.com/prestosql/presto-hive-apache/pull/17

Reply:
Seems has been released ( 3.1.2-4)